src/docs_website/src/html/page.html

```diff
@@ -97,6 +97,7 @@
     <div class="resizer"></div>
     <main>
       <article>
+        <aside id="note-nested-css">Note: this site uses nested CSS selectors. The future is already here, it's just not evenly distributed yet! Support the latest version of your browser, which will render these properly.</aside>
```

Comment:
Hm, let's maybe tone down a joke here a bit, a people with older browsers might be feeling somewhat dejected, and we should sweeten the pill. 

So I suggest going for the neutral:

> Note: this site uses nested CSS selectors. Please upgrade your browser or use the reader mode.